public/json/synoptic.json

```diff
@@ -2,15 +2,15 @@
   "type": "display",
   "position": "relative",
   "width": "600px",
-  "height": "200px",
+  "height": "210px",
```

Comment:
Was this a very important 10 pixels? :sweat_smile:

Reply:
Yup.

Reply:
:+1: